app/views/insured/families/_inbox.html.erb

```diff
@@ -1,64 +1,128 @@
-<div class="module inbox col-xs-12">
-  <div class="row">
-    <div class="inboxes" id="inbox_form">
-      <div class="pull-right">
-        <a href="<%= inbox_insured_families_path(@provider) %>">
-          <% if @folder == 'Inbox' %>
-            <strong><%= l10n("inbox") %></strong>
-          <% else %>
-            <%= l10n("inbox") %>
+<% if @bs4 %>
+  <% sorted_inbox_messages = @provider.inbox.messages.select{|m| @folder == (m.folder.try(:capitalize) || 'Inbox') }.sort_by(&:created_at).reverse %>
+  <div id="inbox-headers">
```

Comment:
add class of 'tab-holder'

Reply:
https://github.com/ideacrew/enroll/pull/4031/commits/2acbaa0d22ab53f4730d06651ec91c91b1a69c73#diff-02e0e47ade8e44ae0909e3b718d6a44798b9ba6e661c34186c82b4997b36705cR3